packages/vats/test/bootstrapTests/test-vaults-integration.js

```diff
@@ -182,15 +202,17 @@ test('close vault', async t => {
     {
       offerId: 'close-well',
       collateralBrandKey,
-      giveMinted: 5.0,
```

Comment:
this was a bug in the test. it wasn't detected because I mistakenly thought `numWantsSatisified: 1` meant it succeeded. There were no wants to they were trivially satisfied.

That it _looked like_ it succeeded and it didn't underlines the value of the promise rejecting.